src/utils/upload/processUpload.js

```diff
@@ -217,16 +216,16 @@ const processUpload = async (filesList, sampleType, samples, activeProjectUuid,
   });
 };
 
-const bundleToFile = async (bundle, technology) => {
+const fileObjectToFileRecord = async (fileObject, technology) => {
```

Comment:
`bundleToFile` wasn't very self explanatory, but `fileObjectToFileRecord` isn't very explanatory either, what do you think of perhaps coming up with a better name or adding comments explaining what is the input and what is the output?

Example of comments.

  ```
/**
   * Does cool things.
   * @param {string} fileObject blalbalba.
   * @param {string} technology blsblslfs.
   * @returns {object} fileRecord blablabladblsal
   */
   
```

Reply:
I'll add the comment